layouts/_default/cve-feed.rss.xml

```diff
@@ -42,6 +42,7 @@
       <pubDate>{{ time.Format "Mon, 02 Jan 2006 15:04:05 -0700" .date_published | safeHTML }}</pubDate>
       <guid>{{ .external_url }}</guid>
       <description>{{ htmlEscape .summary }}</description>
+      <status>{{ .status }}</status>
```

Comment:
There's no `<status>` element in Atom. Try:
```suggestion
      <category term="{{ .status }}" scheme="https://k8s.io/2024-06-27/cve-item-status" />
```

(I minted this scheme as an example)

Reply:
Sure! I'm not sure I understand what the _scheme_ field should actually contain though. Would you mind clarifying that for me?

Reply:
It needs to be a URI, a bit like a relationship in the semantic web.https://graphdb.ontotext.com/documentation/9.5/free/introduction-to-semantic-web.html explains more but is complex to learn.

If you're not sure, go with the suggestion.

Reply:
Done then :)